src/main/drivers/max7456.c

```diff
@@ -75,29 +75,39 @@ static void max7456_send_dma(void* tx_buffer, void* rx_buffer, uint16_t buffer_s
 #endif
 
     // Common to both channels
+    DMA_StructInit(&DMA_InitStructure);
```

Comment:
this seemed to fix the initialization. otherwise it just hung waiting for a response